public-api/src/main/resources/public-api.yaml

```diff
@@ -529,6 +570,60 @@ definitions:
         items:
           $ref: "#/definitions/ConceptAnalysis"
 
+  SurveyDemographicAnalysis:
+    type: object
+    required:
+      - items
+    properties:
+      genderAnalysis:
+        description: gender analysis
+        type: object
+        $ref: "#/definitions/Analysis"
+      ageAnalysis:
+        description: age analysis
+        type: object
+        $ref: "#/definitions/Analysis"
+      raceAnalysis:
+        description: race analysis
+        type: object
+        $ref: "#/definitions/Analysis"
+      ethnicityAnalysis:
```

Comment:
add measurementValueMale, measurementValueFemale here and we can get rid of measurement api :)

Reply:
and measurementValueAgeAnalysis ...

Reply:
Okay. Added everything to concept analysis and it seems to work fine.